xDripG5/Transmitter.swift

```diff
@@ -8,14 +8,23 @@
 
 import Foundation
 import CoreBluetooth
+import HealthKit
 import os.log
 
 
 public protocol TransmitterDelegate: class {
+    func dequeuePendingCommand(for transmitter: Transmitter) -> Command?
```

Comment:
These `Command` methods aren't quite like the other delegate methods, and I think they should probably be defined in a separate protocol, e.g. "`TransmitterCommandSource`" and as a separate property on `Transmitter`, e.g. `weak var commandSource: TransmitterCommandSource?`

They also need clear documentation about their concurrency, which is discussed in comments below.